### PR TITLE
OADP-2175: Make use of restic secret to pass datamover retain policy instead of datamover config map

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**
Related to https://github.com/openshift/oadp-operator/pull/1078

### CODE REVIEW

File: controllers/cleanup.go [hunk 1]
```diff
@@ -95,21 +95,23 @@ func (r *VolumeSnapshotBackupReconciler) CleanBackupResources(log logr.Logger) (
 }
 
 func (r *VolumeSnapshotBackupReconciler) isRetainPolicySet(vsb *volsnapmoverv1alpha1.VolumeSnapshotBackup) (bool, error) {
-	cm, err := GetDataMoverConfigMap(vsb.Spec.ProtectedNamespace, vsb.Status.SourcePVCData.StorageClassName, r.Log, r.Client)
-	if err != nil {
+	// change this logic to check for secret
```
Comment (kaovilai) [inline controllers/cleanup.go:98]: Still WIP?

Comment (shubham-pampattiwar) [reply controllers/cleanup.go]: PR not in WIP, removed the comment

**Review by kaovilai — COMMENTED** [review]
/approve

**Pull request merged**